chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java

```diff
@@ -120,7 +121,7 @@ public void onLambdaSuccess(JSONObject result){
                     public void onLambdaFail(Error error) {
 
                         if (callback != null) {
-                            callback.onFail(error.getMessage());
+                            callback.onFail(error);
```

Comment:
I think it won't be better if we could have something similar to `ConversationNotFoundError` that user can get the conversation directly? Open the discuss

Reply:
won't be better or would be better?

Reply:
would be better :P